src/Fantomas/FormatConfig.fs

```diff
@@ -9,6 +9,23 @@ type FormatException(msg: string) =
 
 type Num = int
 
+type MultilineFormatterType =
+    | CharacterWidth
+    | LogicalSize
+    static member toConfigString =
```

Comment:
Nitpick: should have a capital letter I think. Did you ran fsharplint on this?
This is not yet part of our CI system but you can restore the dotnet tools (`dotnet tool restore`)
And run `dotnet fsharplint lint fantomas.sln`.

Reply:
I had not run the linter on it, but even when I did there was no warning associated with these two.  I think it thinks it is okay because it is a `static` member, so is treated the same as a module-local function, and so can be camelCase.

Would you like me to change it anyways?

Reply:
Yes please, the linter doesn't mention it as I was expecting but all the other `static member` occurrences are uppercased.

Reply:
Will do.